app/controllers/landable/public/sitemap_controller.rb

```diff
@@ -8,7 +8,7 @@ def index
           host: Landable.configuration.sitemap_host || request.host,
           protocol: Landable.configuration.sitemap_protocol,
           exclude_categories: Landable.configuration.sitemap_exclude_categories,
-          sitemap_additional_paths: Landable.configuration.sitemap_additional_paths,
```

Comment:
BOO RuboCop, I am a fan of trailing commas as it makes it easier whenever you need to add on later